### ISSUE TITLE
Fix: Images without alt attributes affecting accessibility

### ISSUE THREAD

**Issue by Fireentity** [layer5io/layer5#6829]
### Description
On the Layer5 website hompage: [layer5io/](https://layer5.io/), two images are missing the alt attribute, as reported by PageSpeed Insights during an accessibility audit. This affects accessibility (screen readers cannot describe the images) and SEO.

### Expected Behavior
All informative images should have meaningful alt attributes (e.g., "Amazon Web Services Logo", "Google Cloud Logo"). Decorative images should explicitly use an empty alt=""

### Screenshots

<img width="966" height="415" alt="Image" src="https://github.com/user-attachments/assets/eec6af1f-f5c9-40a7-b608-8e2d15cf1b41" />

### Environment:
This issue occurs across all environments and browsers, since it originates from the website’s codebase and is not client-dependent.

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by Fireentity** [comment]
Hi, please assign this issue to me, my development environment is working correctly and I will proceed with adding the appropriate alt attributes to the affected images, as suggested in the description.

**Comment by Divyarora0906** [comment]
@blake @thrawn01 @dudash 
Hi 👋, I’d like to work on this issue.  
I will add meaningful alt attributes to the missing images (e.g., "Amazon Web Services Logo", "Google Cloud Logo") to improve accessibility and SEO, as described in the issue.  

Could you please assign this to me? 🙂

**Comment by miacycle** [comment]
Thanks, @Fireentity! 👍
